HunterPie/address/MonsterHunterWorld.421470.map

```diff
@@ -3,6 +3,7 @@ Address BASE 0x140000000
 Address CANTEEN_OFFSET 0x04FE79C0
 Address MONSTER_SELECTED_OFFSET 0x04FFF768
 Address PLAYER_DATA_OFFSET 0x04FFF768
+Address NOOB_OFFSET 0x506E340
```

Comment:
Please, make the name more descriptive, "NOOB_OFFSET" is too vague and this pointer definitely doesn't point to a "NOOB" and I don't even know what exactly it's pointing to by looking at it's name.

Reply:
sure. I'll see what I can come up with. idk exactly what it points to though, but it looks like it's checking if you can even harvest.

Reply:
I removed it as mentioned below. lmk what you think